stories/carousel/carousel.stories.js

```diff
@@ -0,0 +1,480 @@
+import '../../dist/carousel.css';
+
+export default {
+    title: 'Components/Carousel',
+    parameters: {
+        description: `The carousel component is used for browsing through a set of images. One or several items can be displayed at a time. The user can navigate to either the next or the previous item.
+
+## Usage
+
+- to achieve a strong visual representations of the items you want to display
+- to display items sequentially or side by side
+
+## Responsiveness
+
+The size of the content area is adjusted automatically, depending on the screen and the available space.
+On non-touchable devices, the buttons on the left and right of the page indicator or the content are used for navigating to the next or the previous item/s. On touchable devices the navigation is achieved with a swipe gesture and the buttons are not displayed. 
+The page indicator wraps on the next line if the content is too long.
+
+## Layout
+
+- **Content**- (mandatory) the area displaying either the current item or a set of items.
+- **Page indicator**- (optional) can be above or below the content area. After the pages reach the count of 8, the dots are replaced by numeral representation (valid for all devices).
+The numeral format is the following: **1 of 9**
+- **Buttons**- on non-touch devices, paging buttons either float above the left and right sides of the content area, or appear in the paging indicator area. The buttons are not displayed on touchable devices. The <code class="docs-code">fd-carousel--no-navigation</code> modifier class applied with <code class="docs-code">fd-carousel</code> when the device is touchable will hide the buttons.
+
+## Behavior and Interaction
+
+- **Navigation for single items**- when the user navigates from the current item to another item, the current item is moved out of the content area, and the next or previous item slides in (depending on the direction of navigation). If the item set contains only one item, navigation is deactivated.
+- **Navigation for single items**- when the user clicks one of the paging buttons, the rightmost or leftmost item is moved out of the content area, and the next or previous item slides in (depending on the navigation direction).
+- **Looping**- if looping is enabled, the carousel jumps back to the first item once all items have been displayed.
+- **Paging**- the current position inside the set of items is displayed using an optional paging indicator. 
+
+## Empty State
+
+When no pages are loaded, the Carousel displays a MessagePage component with <code class="docs-code">sap-icon--document</code> icon.
+
+
+## Accessibility
+
+The div element with class <code class="docs-code">fd-carousel</code> has a visually hidden sibling div element with <code class="docs-code">role="region"</code> and <code class="docs-code">aria-live="polite"</code>. Its text should be dynamically updated using JavaScript so that content changes of the Carousel are announced to the user.
```

Comment:
Yes, a screen reader

Reply:
Okay, then this is what I would add at the end: "This way, the text will be dynamically updated using Javascript so that carousel's content changes can be announced to the user by screen readers."